src/components/Common/EmojiPickerIconButton.tsx

```diff
@@ -1,7 +1,7 @@
-import { Box, IconButton } from "@mui/material";
 import EmojiEmotionsOutlinedIcon from "@mui/icons-material/EmojiEmotionsOutlined";
-import React, { useState, useEffect, useRef } from "react";
+import { Box, IconButton } from "@mui/material";
 import EmojiPicker, { EmojiClickData, EmojiStyle } from "emoji-picker-react";
+import React, { useState, useEffect, useRef } from "react";
```

Comment:
Do we care if the imports within the objects are also sorted alphabetically?

Reply:
I don't think the plugin supports that, so unfortunately we might have to leave it like that lol. Feel free to browse their docs, maybe I missed it https://github.com/un-ts/eslint-plugin-import-x